### DISCUSSION TITLE
supporting for loading ui framework in AppWindow

### DISCUSSION THREAD

**Discussion by oldoldman** [microsoft/WindowsAppSDK#3603] [General]
Hi guys,
From this [link](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/windowing/windowing-overview#limitations) loading framework ui tree is not supported for now, is there any plan to do that? Many thanks!

**Comment by DarranRowe** [comment]
There are plans.
If you look in the experimental releases of the Windows App SDK, with 1.3 experimental being the latest, you will find the latest snapshot of their plans. The namespaces that you want are Microsoft.UI.Content and Microsoft.UI.Xaml.Hosting.
